csharp-intermediate-topics/DifferencesRecordStructAndRecordClassInCSharp/DifferencesRecordStructAndRecordClassInCSharp/Program.cs

```diff
@@ -0,0 +1,23 @@
+﻿using DifferencesRecordStructAndRecordClassInCSharp;
+
+var recordClassDefaultValue = DefaultValues.GetRecordClassDefaultValue();
+var recordStructDefaultValue = DefaultValues.GetRecordStructDefaultValue();
+Console.WriteLine($"Record class default value is {(recordClassDefaultValue == null ? "null" : recordClassDefaultValue)}");
+Console.WriteLine($"Record struct default value is {(recordStructDefaultValue == null ? "null" : recordStructDefaultValue)}");
+
+public record struct PersonRecordStruct(string Name, DateTime BirthdayDate, int YearsOld, string[] PhoneNumbers);
```

Comment:
If you ask me, I would like to see the struct/class types in separate files, what do you think?

Reply:
Yes, makes perfect sense. Moved it.